templates/firstboot/mellanox_fw_update.yaml

```diff
@@ -144,7 +152,7 @@ resources:
                         if not line:
                             continue
                         dev = line.split()[0]
-                        if dev.endswith('.0'):
+                        if dev.endswith('.0') and (not _DEV_WHITE_LIST or dev in _DEV_WHITE_LIST):
```

Comment:
why _DEV_WHITE_LIST is global and not passed in the MlnxDevices init

Reply:
@moshe010 Done